model/fv_nesting.F90

```diff
@@ -2573,14 +2447,18 @@ subroutine twoway_nest_update(npx, npy, npz, zvir, ncnst, sphum,     &
               neststruct%isu, neststruct%ieu, neststruct%jsu, neststruct%jeu, &
               npx, npy, npz, 0, 0, &
               neststruct%refinement, neststruct%nestupdate, upoff, 0, &
-              neststruct%parent_proc, neststruct%child_proc, parent_grid, grid_number-1)
+              parent_grid%neststruct%parent_proc, neststruct%child_proc, parent_grid, grid_number-1)
 
       endif !conv_theta
 
       call mpp_sync!self
```

Comment:
Do you want to perform an mpp_sync_self or is the full sync necessary here and down below the other update_coarse_grid?